Cargo.toml

```diff
@@ -61,3 +61,6 @@ ethers = { version = "2.0.4", features = ["rustls"] }
 
 [patch.crates-io]
 sha3 = { git = "https://github.com/RustCrypto/hashes", tag = "sha3-v0.10.6" }
+
+[profile.dev]
+debug = 0
```

Comment:
I was unaware that this was being used, given that it has been disabled on other matter-labs repositories with a note that it can be enabled locally when needed. I only encountered this as the binary goes OOM during linking phase with `mold` and `32GB` ram on my Ubuntu.

Reply:
As this is causing OOM, we can disable. But let's update all debug launch configurations within `.vscode/launch.json` to set the following environment variable values so developers don't have to randomly remember to change this value when attempting to debug locally in VSCode:
```json
        {
            "name": "Debug era_test_node",
            // ...
            "env": {
                "CARGO_PROFILE_DEV": "true"
            },
            // ...
        }
```